rwgame/states/IngameState.cpp

```diff
@@ -351,18 +351,7 @@ const ViewCamera& IngameState::getCamera(float alpha) {
             glm::normalize(glm::cross(glm::vec3(0.f, 0.f, 1.f), direction));
         auto up = glm::normalize(glm::cross(direction, right));
 
-        glm::mat3 m;
-        m[0][0] = direction.x;
-        m[0][1] = right.x;
-        m[0][2] = up.x;
-
-        m[1][0] = direction.y;
-        m[1][1] = right.y;
-        m[1][2] = up.y;
-
-        m[2][0] = direction.z;
-        m[2][1] = right.z;
-        m[2][2] = up.z;
+        glm::mat3 m(direction, right, up);
```

Comment:
This change totally jumbles the camera angles

Reply:
Oops, screw up on my part!